### DISCUSSION TITLE
Componente Table com Actions no subtitle

### DISCUSSION THREAD

**Discussion by leonardoquintania** [po-ui/po-angular#844] [Tire suas dúvidas]
Boa tarde,

Gostaria de saber se é possível adicionar um conjunto de ações no subtitle quando possuo um componente com detail.

**Comment by jhonyeduardo** [comment]
Olá @leonardoquintania 

Você possui algum mockup/imagem para dar um entendimento maior?

Caso for adicionar ação no PoTableSubtitleColumn atualmente não é possível.
